src/PrestaShopBundle/Resources/config/routing/admin/sell/catalog/products_v2/product.yml

```diff
@@ -331,7 +332,7 @@ admin_products_bulk_disable_shop_group:
     _controller: PrestaShopBundle\Controller\Admin\Sell\Catalog\Product\ProductController::bulkDisableShopGroupAction
     _legacy_controller: AdminProducts
     _legacy_link:
-      - AdminProducts:submitBulkenableSelectionproduct
+      - AdminProducts:submitBulkdisableSelectionproduct
```

Comment:
Maybe we can simplify this:
`_legacy_link: AdminProducts:submitBulkdisableSelectionproduct`